### PR TITLE
When the TTSJobs site is updated, also update the Handbook if appropriate

### PR DESCRIPTION
**Pull request by mgwalker**
Changes proposed in this pull request:
- When the TTSJobs site is updated...
  - copies the Handbook repo
  - check is the jobs data in TTSJobs is different from the Handbook
  - if so, makes the changes in the Handbook
  - creates a pull request in the Handbook repo with the changes

### CODE REVIEW

File: .github/workflows/update_handbook.yaml
Comment (alexsobledotgov) [inline .github/workflows/update_handbook.yaml:10]: Typo: "update TTS Handbook"

**Comment by alexsobledotgov** [comment]
@mgwalker How do you think we should test this? We could wait until there's an actual external job posting that needs to be synced, but my preference would be to test with some fake data first. 

Once https://github.com/18F/handbook/pull/3460 is merged, could we do something like this?:

1. Put some test jobs data on this branch.
2. Trigger a workflow on this branch to open a "Do not merge -- testing" PR into the Handbook.
3. Check the preview build on the Handbook PR.
4. Revert the test data on this branch back to an empty array.

File: .github/workflows/update_handbook.yaml
Comment (mgwalker) [inline .github/workflows/update_handbook.yaml:10]:
```suggestion
    name: update TTS Handbook
```

**Comment by mgwalker** [comment]
No, that won't work without changing the code here. This action only runs after merging to the main branch. My goal is for this repo's main branch to be the source of truth for job postings that are open to the public.

But lucky for both of us, before I made it run on main, I first had it run on this branch, and it produced this handy-dandy PR on the Handbook: https://github.com/18F/handbook/pull/3459

**Comment by alexsobledotgov** [comment]
@mgwalker Alrighty, well — there's our test run PR! And our test data! 👍

**Review by alexsobledotgov — APPROVED** [review]

**Pull request merged**
